StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkStepUpVerification/NetworkingLinkStepUpVerificationDataSource.swift

```diff
@@ -67,7 +67,8 @@ final class NetworkingLinkStepUpVerificationDataSourceImplementation: Networking
         return apiClient.selectNetworkedAccounts(
             selectedAccountIds: selectedAccountIds,
             clientSecret: clientSecret,
-            consumerSessionClientSecret: consumerSession.clientSecret
+            consumerSessionClientSecret: consumerSession.clientSecret,
+            consentAcquired: nil
```

Comment:
this NULL is fine; there will be a separate PR that covers the "step up verification case"